### PR TITLE
feat: OPX-3.2

### PR DESCRIPTION
**Pull request by Tejaswi-Goel**
* Update: dequeue object from cps publish queue if address not in connection cache
* Update: initializing variable for coverity fix
* Update: fix coverity issue regarding uninitialized variable

Signed-off-by: Tejaswi Goel <Tejaswi_Goel@Dell.com>

### CODE REVIEW

**Comment by Tejaswi-Goel** [comment]
Pull-request updated, HEAD is now f856f797ab314c07e7d045a9eb7d7ff7f2ccba49

**Comment by Tejaswi-Goel** [comment]
Pull-request updated, HEAD is now 610ba0e9ba66841a355a69ecbc4eec340f9f63ff

**Comment by Tejaswi-Goel** [comment]
Pull-request updated, HEAD is now 2d99b0485200cc782f39444c61e2666f50c7fca7

File: Makefile.am [hunk 1]
```diff
@@ -75,7 +72,7 @@ libopx_cps_api_common_la_SOURCES = \
           src/non-core/cps_api_db_interface_tools.cpp
 
 
-#Unit tests that will be merged into the config file at a later date
+#Unit tests that will be merged into the config file at a later daye
```
Comment (rakeshdatta) [inline Makefile.am]: remove this change

**Review by rakeshdatta — CHANGES_REQUESTED** [review]
Resolve the comments

**Comment by Tejaswi-Goel** [comment]
Pull-request updated, HEAD is now c759c6c8effe4344a00834ea0bbf67a48a402e5e

File: Makefile.am (hunk 1 above)
Comment (Tejaswi-Goel) [reply Makefile.am]: done

**Comment by Tejaswi-Goel** [comment]
Pull-request updated, HEAD is now 4cf9483699713be1309cf4aa7794348394bbb83d

**Review by rakeshdatta — APPROVED** [review]

**Pull request merged**